tyk-headless/templates/secrets.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secrets
```

This should be more in line with the rest of the resources.
```suggestion
  name: secrets-{{ include "tyk-headless.fullname" . }}
```

Same in the other charts.

---

Have changed the name now.